images/encryption/encryption.go

```diff
@@ -389,6 +399,9 @@ func cryptManifestList(ctx context.Context, cs content.Store, desc ocispec.Descr
 		}
 		newManifests = append(newManifests, newManifest)
 	}
+	if cryptoOp == cryptoOpUnwrapOnly {
```

Comment:
should this also check if `newManifests` is not empty?

Reply:
ahhh i see, i misread `if err != nil || cryptoOp == cryptoOpUnwrapOnly` as `if err != nil && cryptoOp == cryptoOpUnwrapOnly`, that was where my confusion was stemming from.